hoomd/hpmc/pytest/test_sphere_union.py

```diff
@@ -0,0 +1,175 @@
+import hoomd
+import hoomd.hpmc
+from hoomd.hpmc import _hpmc
+import pytest
+import copy
+
+sph_args_1 = {'diameter': 1, 'orientable': 0, 'ignore_statistics': 1}
+sph_args_2 = {'diameter': 9, 'orientable': 1, 'ignore_statistics': 1}
+sph_args_3 = {'diameter': 4, 'orientable': 0, 'ignore_statistics': 0}
+
+sph_union_args1 = {'shapes': [sph_args_1, sph_args_2],
+                   'positions': [(0, 0, 0), (0, 0, 1)],
+                   'orientations': [(1, 0, 0, 0), (1, 1, 0, 0)],
+                   'overlap': [1, 1],
+                   'capacity': 4,
+                   'ignore_statistics': 1}
+
+sph_union_args2 = {'shapes': [sph_args_1, sph_args_3],
+                   'positions': [(1, 0, 0), (0, 0, 1)],
+                   'orientations': [(1, 1, 0, 0), (1, 0, 0, 0)],
+                   'overlap': [1, 2],
+                   'capacity': 3,
+                   'ignore_statistics': 1}
+
+sph_union_args3 = {'shapes': [sph_args_3, sph_args_2],
+                   'positions': [(1, 1, 0), (1, 0, 1)],
+                   'orientations': [(1, 0, 0, 0), (1, 0, 0, 0)],
+                   'overlap': [1, 0],
+                   'capacity': 2,
+                   'ignore_statistics': 0}
+
+sph_union_args4 = {'shapes': [sph_args_1, sph_args_2, sph_args_3],
+                   'positions': [(0, 0, 0), (0, 1, 1), (1, 1, 1)],
+                   'orientations': [(1, 1, 1, 0), (1, 1, 0, 0), (1, 0, 0, 1)],
+                   'overlap': [1, 1, 1],
+                   'capacity': 4,
+                   'ignore_statistics': 1}
+
+
+def test_dict_conversions():
+
+    test_sph_union1 = _hpmc.SphereUnionParams(sph_union_args1)
+    test_sph_dict1 = test_sph_union1.asDict()
+    assert test_sph_dict1 == sph_union_args1
+
+    test_sph_union2 = _hpmc.SphereUnionParams(sph_union_args2)
+    test_sph_dict2 = test_sph_union2.asDict()
+    assert test_sph_dict2 == sph_union_args2
+
+    test_sph_union3 = _hpmc.SphereUnionParams(sph_union_args3)
+    test_sph_dict3 = test_sph_union3.asDict()
+    assert test_sph_dict3 == sph_union_args3
+
+    test_sph_union4 = _hpmc.SphereUnionParams(sph_union_args4)
+    test_sph_dict4 = test_sph_union4.asDict()
+    assert test_sph_dict4 == sph_union_args4
+
+
+def test_shape_params():
+
+    mc = hoomd.hpmc.integrate.SphereUnion(23456)
+
+    mc.shape['A'] = dict()
+    print(mc.shape['A'])
```

Comment:
Another `print` statement here. Please do a grep across the testing directory and remove these.